CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 ### Added
 - fromScope, uses forPageAfterId.
 - a new option "--parallel" for ImportCommand, can only be used with [Trackable-Jobs](https://github.com/mateusjunges/trackable-jobs-for-laravel) package currently.
+- ElasticParams trait, that adds 'getElasticsearchScore' and 'getElasticsearchHighlight' to the model after performing a search.
```

Comment:
it should be a new version 
8.0.0-alpha.2 

or if you want to have it in stable, you should to create a PR on top of master branch

then it will be 7.7.0

Reply:
Updated Changelog.md